### DISCUSSION TITLE
Does anyone know if there is an eslint rule for static functions?

### DISCUSSION THREAD

**Discussion by ingrocha** [eslint/eslint#16572] [Q&A] [answered]
Hello everyone; I am looking for a rule to validate if all functions in a class have the static property. Can anyone help me?

### Bad
``` typescript
function foo() {
...
}
```

### Correct
``` typescript
static function foo () {
...
}
```

**Comment by aladdin-add** [comment, marked as answer]
no such a rule in eslint core. you'll have to look for 3rd-party plugins or write your own.
